econml/tests/test_inference.py

```diff
@@ -93,15 +261,16 @@ def test_degenerate_cases(self):
             pop.print()  # verify that we can access all attributes even in degenerate case
 
     def test_can_summarize(self):
-        LinearDMLCateEstimator().fit(
+        LinearDMLCateEstimator(model_t=LinearRegression(), model_y=LinearRegression()).fit(
```

Comment:
What was wrong with just using the defaults here?

Reply:
Just faster ;)